DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,13 @@ public function getConfigTreeBuilder()
         $rootNode->setBuilder(new AllowExtraPropertiesNodeBuilder());
 
         $rootNode->children()
+            ->arrayNode('allowedParamConverters')
+                ->prototype('array')
+                    ->children()
+                        ->scalarNode('name')->isRequired()->end()
```

Comment:
On first glance, it seems this does not match your sample config in https://github.com/kfzteile24/payment-service/pull/185/files#diff-5270406e7d2dfa23df27a5555d97f7a8R142 , which has an additional `json_schema` key.

Also, do we need this to be _required_?